app/javascripts/main/trayManager.js

```diff
@@ -42,41 +73,23 @@ export class TrayManager {
       tray.setContextMenu(trayContextMenu);
     };
 
-    const trayContextMenu = Menu.buildFromTemplate([{
-      id: 'ShowWindow',
-      label: 'Show',
-      click: tray.toggleWindowVisibility.bind(this, true)
-    }, {
-      id: 'HideWindow',
-      label: 'Hide',
-      click: tray.toggleWindowVisibility.bind(this, false)
-    },
-    {
-      type: 'separator'
-    },
-    {
-      id: 'quit',
-      label: 'Quit',
-      click: app.quit.bind(app)
-    }]);
-
-    tray.setToolTip('Standard Notes');
+    tray.updateContextMenu();
+    tray.setToolTip("Standard Notes");
     tray.setContextMenu(trayContextMenu);
 
-    tray.on('click', () => {
-      tray.popUpContextMenu();
-    });
+    if (this.platform === Platforms.Windows) {
+      tray.on("click", () => tray.toggleWindowVisibility(true));
+    }
 
-    this.bindListeners(tray);
+    this.window.on("hide", tray.updateContextMenu);
+    this.window.on("focus", tray.updateContextMenu);
+    this.window.on("blur", tray.updateContextMenu);
   }
 
-  bindListeners(tray) {
-    this.window.on('hide', () => {
-      tray.updateContextMenu();
-    });
-
-    this.window.on('focus', () => {
-      tray.updateContextMenu();
-    });
+  removeTrayIcon() {
```

Comment:
Good to mark this with:

```
/** @public */
removeTrayIcon() {
```

so it's clear it's being accessed from the outside. Because at first I thought this was an internal method. And likewise for internal methods:

```
/** @private */
```

In this case it looks like all the tray's methods are public, so we could annotate all of them as such.